src/gui/MainWindow.cpp

```diff
@@ -782,6 +786,9 @@ void MainWindow::restoreWidgetState( QWidget * _w, const QDomElement & _de )
 		flags = _de.attribute( "maximized" ).toInt() ?
 				( flags | Qt::WindowMaximized ) :
 				( flags & ~Qt::WindowMaximized );
+		flags = _de.attribute( "fullscreen" ).toInt() ?
+				( flags | Qt::WindowFullScreen ) :
+				( flags & ~Qt::WindowFullScreen );
```

Comment:
Is this necessary? Not exactly sure what `restoreWidgetState()` does.

Reply:
I'm pretty sure this is used to put widgets in the correct (saved) position when loading a file. Considering that and the fact that individual widgets don't have a fullscreen attribute, I would remove these lines. Unless, of course, your testing shows that it breaks something to do so.

Reply:
In fact, I don't think fullscreen state should be restored on file load at all, so there's another argument for not saving or loading this attribute anywhere.